src/index.js

```diff
@@ -0,0 +1,39 @@
+// ----------------- Import Required Elements
```

Comment:
- [ ] Please, make sure that you implemented a function for editing task descriptions 👍 This is the milestone requirement.